webapi/src/main/scala/org/knora/webapi/slice/resources/repo/service/value/queries/CreateValueQueryBuilder.scala

```diff
@@ -0,0 +1,766 @@
+import org.eclipse.rdf4j.model.vocabulary.XSD
+import org.eclipse.rdf4j.sparqlbuilder.constraint.Bind
+import org.eclipse.rdf4j.sparqlbuilder.constraint.Expressions
+import org.eclipse.rdf4j.sparqlbuilder.constraint.SparqlFunction
+import org.eclipse.rdf4j.sparqlbuilder.constraint.propertypath.builder.PropertyPathBuilder
+import org.eclipse.rdf4j.sparqlbuilder.core.SparqlBuilder
+import org.eclipse.rdf4j.sparqlbuilder.core.SparqlBuilder.`var` as variable
+import org.eclipse.rdf4j.sparqlbuilder.core.Variable
+import org.eclipse.rdf4j.sparqlbuilder.core.query.Queries
+import org.eclipse.rdf4j.sparqlbuilder.graphpattern.GraphPattern
+import org.eclipse.rdf4j.sparqlbuilder.graphpattern.GraphPatterns
+import org.eclipse.rdf4j.sparqlbuilder.graphpattern.TriplePattern
+import org.eclipse.rdf4j.sparqlbuilder.rdf
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf.iri
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf.literalOf
+import org.eclipse.rdf4j.sparqlbuilder.rdf.Rdf.literalOfType
+import org.eclipse.rdf4j.sparqlbuilder.rdf.RdfObject
+import org.eclipse.rdf4j.sparqlbuilder.rdf.RdfPredicate
+import org.eclipse.rdf4j.sparqlbuilder.rdf.RdfSubject
+
+import java.time.Instant
+import java.util.UUID
+import scala.util.chaining.scalaUtilChainingOps
+
+import dsp.valueobjects.UuidUtil
+import org.knora.webapi.messages.OntologyConstants
+import org.knora.webapi.messages.SmartIri
+import org.knora.webapi.messages.StringFormatter
+import org.knora.webapi.messages.twirl.SparqlTemplateLinkUpdate
+import org.knora.webapi.messages.twirl.queries.sparql
+import org.knora.webapi.messages.v2.responder.standoffmessages.StandoffTagAttributeV2
+import org.knora.webapi.messages.v2.responder.valuemessages.ValueContentV2
+import org.knora.webapi.slice.common.domain.InternalIri
+import org.knora.webapi.slice.common.repo.rdf.Vocabulary.KnoraBase as KB
+import org.knora.webapi.store.triplestore.api.TriplestoreService.Queries.Update
+
+object CreateValueQueryBuilder {
+  private[service] def createValueQueryTwirl(
+    dataNamedGraph: InternalIri,
+    resourceIri: InternalIri,
+    propertyIri: SmartIri,
+    newValueIri: InternalIri,
+    newValueUUID: UUID,
+    value: ValueContentV2,
+    linkUpdates: Seq[SparqlTemplateLinkUpdate],
+    valueCreator: InternalIri,
+    valuePermissions: String,
+    creationDate: Instant,
+  )(implicit sf: StringFormatter): Update = {
+    val twirlSparql = sparql.v2.txt
+      .createValue(
+        dataNamedGraph = dataNamedGraph.value,
+        resourceIri = resourceIri.value,
+        propertyIri = propertyIri,
+        newValueIri = newValueIri.value,
+        newValueUUID = newValueUUID,
+        value = value,
+        linkUpdates = linkUpdates,
+        valueCreator = valueCreator.value,
+        valuePermissions = valuePermissions,
+        creationDate = creationDate,
+        stringFormatter = sf,
+      )
+      .toString
+      .strip
+
+    Update(twirlSparql)
+  }
```

Comment:
is this still in use? or can it be removed, together with the twirl template?

Reply:
Removed, all converted to golden.